ngraph/frontend/tensorflow/include/tensorflow_frontend/frontend.hpp

```diff
@@ -5,18 +5,11 @@
 #pragma once
 
 #include <frontend_manager/frontend.hpp>
+#include <frontend_manager/input_model.hpp>
 #include <tensorflow_frontend/model.hpp>
 #include <tensorflow_frontend/utility.hpp>
 
-using namespace ngraph::frontend;
-
-namespace tensorflow {
-class GraphDef;
-class NodeDef;
-namespace ngraph_bridge {
-class GraphIteratorProto;
-}
-}  // namespace tensorflow
+using namespace ::ngraph::frontend;
```

Comment:
"using" in the header file?
not sure if this is ok, may be the cause of the name collision

Reply:
done.